initfiles/componentfiles/configxml/buildsetCC.xml.in

```diff
@@ -162,6 +162,10 @@
                           path="FileDkcAccess"
                           resource="FileDkcAccess"
                           service="ws_fs"/>
+     <AuthenticateFeature description="Access to upload files to LZ"
```

Comment:
Should we spell out "Landing Zone"  ( see the feature right below this, it reads "Access to files in dropzone")